projects/ngx-formentry/src/form-entry/helpers/js-expression-helper.ts

```diff
@@ -164,6 +166,68 @@ export class JsExpressionHelper {
     return height && weight && refSectionObject ? formattedSDValue : null;
   }
 
+  getAgePosition(age) {
```

Comment:
This stuff was already covered by the functions I gave in the ticket. If you want to invert the index, that's fine, but then the age index, for instance, is just 6 - the calculated index, etc.

Note that some of your functions here are just wrong. There are only 5 bins for systolic blood pressure, for example.

Reply:
okay, I will revert to my original formulas you shared, I just wanted to simplify them first and see if I can get an output but now I have a better understanding and I can follow your provided binaries.